public/pages/template.html

```diff
@@ -4,37 +4,30 @@
 <h1 class="cover-heading">
     {{account}}/{{repo}}
 </h1>
-<p class="lead">
+<p class="lead badges">
     <a href="https://gitlicense.com/license/{{account}}/{{repo}}/" style="margin: auto;">
         <img src="https://gitlicense.com/badge/{{account}}/{{repo}}/" />
         <img src="https://gitlicense.com/badge/{{account}}/{{repo}}?color=red" />
         <img src="https://gitlicense.com/badge/{{account}}/{{repo}}?color=blue" />
     </a>
 </p>
-<h2>Markdown:</h2>
+<h2>Markdown</h2>
 <div class="input-group">
-    <textarea class="form-control" id="md" readonly>
-[![GitLicense](https://gitlicense.com/badge/{{account}}/{{repo}})](https://gitlicense.com/license/{{account}}/{{repo}})
-    </textarea>
+    <input type="text" class="form-control" id="md" value="[![GitLicense](https://gitlicense.com/badge/{{account}}/{{repo}})](https://gitlicense.com/license/{{account}}/{{repo}})" readonly />
     <button class="btn btn-default" data-clipboard-target="#md">
         <i class="fa fa-copy"></i>
     </button>
 </div>
-<h2>reStructuredText:</h2>
+<h2>reStructuredText</h2>
 <div class="input-group">
-    <textarea class="form-control" id="re" readonly>
-.. image:: https://gitlicense.com/badge/{{account}}/{{repo}}
-  :target: https://gitlicense.com/license/{{account}}/{{repo}}
-    </textarea>
+    <input type="text" class="form-control" id="re" value=".. image:: https://gitlicense.com/badge/{{account}}/{{repo}} :target: https://gitlicense.com/license/{{account}}/{{repo}}" readonly />
```

Comment:
There is a reason why I used textarea tag instead of input. If I used input, the copied value will be on only 1 line and rst can not render it. You can see the difference [here](http://rst.ninjs.org/#Li4gaW1hZ2U6OiBodHRwczovL2dpdGxpY2Vuc2UuY29tL2JhZGdlL2R1Y3RoaWVuYnVpOTcvZ2l0bGljZW5zZSA6dGFyZ2V0OiBodHRwczovL2dpdGxpY2Vuc2UuY29tL2xpY2Vuc2UvZHVjdGhpZW5idWk5Ny9naXRsaWNlbnNlCgouLiBpbWFnZTo6IGh0dHBzOi8vZ2l0bGljZW5zZS5jb20vYmFkZ2UvZHVjdGhpZW5idWk5Ny9naXRsaWNlbnNlCiAgOnRhcmdldDogaHR0cHM6Ly9naXRsaWNlbnNlLmNvbS9saWNlbnNlL2R1Y3RoaWVuYnVpOTcvZ2l0bGljZW5zZQogICAg)

Reply:
Okay, I can revert this change if you wish.

Reply:
@ducthienbui97 How would you feel about making the textarea the same height as a text input? That way you keep the line-heights but the button isn't stretched. If not, I'll revert it to the way it was.

Reply:
How's this? ![Fix](https://i.imgur.com/JYxznjS.png)

Reply:
@LuckehPickle  that looks good 👍

Reply:
I know copy button doesn't look  good and that might be the reason why you want to use input. If that's the case, feel free to open a issue about that.

Reply:
For future reference #13